phoenicis-apps/src/main/java/org/phoenicis/apps/RepositoryManager.java

```diff
@@ -57,8 +59,12 @@ public void setOnError(Consumer<Exception> onError) {
         this.onError = onError;
     }
 
-    public Repository cachedRepository() {
-        return this.cachedRepository;
```

Comment:
The name of the attribute should be repository instead of cachedRepository. Theoretically, the manager could work with any kind of repository, could'nt it?